pkg/kubelet/prober/prober.go

```diff
@@ -112,6 +112,9 @@ func (pb *prober) probe(ctx context.Context, probeType probeType, pod *v1.Pod, s
 	if result == probe.Warning {
 		pb.recordContainerEvent(pod, &container, v1.EventTypeWarning, events.ContainerProbeWarning, "%s probe warning: %s", probeType, output)
 		klog.V(3).InfoS("Probe succeeded with a warning", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name, "output", output)
+	} else if result == probe.Unknown {
```

Comment:
should we use `switch` better than `is`?

Reply:
you right ,i will 
its cleaner that way

Reply:
done